### PR TITLE
4620: Display Order Form - Supplier

### PR DESCRIPTION
**Pull request by talawson05**
Tests are currently ignored because the GET summary hasnt been updated to include the new sections yet

[AB#7145](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7145)

### CODE REVIEW

File: src/OrderFormAcceptanceTests.Objects/Pages/OrderForm.cs [hunk 1]
```diff
@@ -26,5 +27,7 @@ public sealed class OrderForm
 		public By OrganisationAddressCounty => CustomBy.DataTestId("organisation-address-county");
 		public By OrganisationAddressPostcode => CustomBy.DataTestId("organisation-address-postcode");
 		public By OrganisationAddressCountry => CustomBy.DataTestId("organisation-address-country");
-	}
+
+
```
Comment (PhysicsJon) [inline src/OrderFormAcceptanceTests.Objects/Pages/OrderForm.cs:31]: Extra blank lines?

File: src/OrderFormAcceptanceTests.TestData/Address.cs [hunk 2]
```diff
@@ -27,7 +27,6 @@ public Address Generate()
             var randomAddress = faker.Address;
             return new Address()
             {
-                AddressId = faker.Random.Number(),
                 Line1 = randomAddress.StreetAddress(),
```
Comment (PhysicsJon) [inline src/OrderFormAcceptanceTests.TestData/Address.cs:30]: Do we use Address.Line2 or Address.Line3 for anything in the tests?

Comment (talawson05) [reply src/OrderFormAcceptanceTests.TestData/Address.cs:30]: not yet, but the application isn't pulling the correct details from OAPI yet, and at that point we might want to store all the lines from the DB and assert they are correct on the UI

Comment (PhysicsJon) [reply src/OrderFormAcceptanceTests.TestData/Address.cs:30]: seems reasonable

**Pull request merged**